src/wallet/rpcwallet.cpp

```diff
@@ -1396,11 +1397,13 @@ UniValue listtransactions(const JSONRPCRequest& request)
             "                                         'send' category of transactions.\n"
             "    \"confirmations\": n,       (numeric) The number of confirmations for the transaction. Negative confirmations indicate the\n"
             "                                         transaction conflicts with the block chain\n"
+            "    \"generated\": xxx,         (bool) Only returns true if transaction is a coinbase one\n"
```

Comment:
The coinbase is the input. The output is simply generated.

Reply:
Doesn't a transaction including a coinbase input qualify as a coinbase one ? We don't enforce this wording strictly if the rest of the codebase I think.